pubtools/_quay/push_docker.py

```diff
@@ -178,6 +178,8 @@ def get_operator_push_items(self):
             if not item.metadata.get("com.redhat.openshift.versions"):
                 msg = "Push item {0} doesn't specify 'com.redhat.openshift.versions'".format(item)
                 raise BadPushItem(msg)
+            if not item.claims_signing_key and docker_push_items:
```

Comment:
What happens when there are no docker push items?

Reply:
If a brew build having only operator item exists, IIB request before signing would fail because bundle to be added is not available.